CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ option(FINUFFT_USE_OPENMP "Whether to use OpenMP for parallelization. If disable
 option(FINUFFT_USE_CPU "Whether to build the ordinary FINUFFT library (libfinufft)." ON)
 option(FINUFFT_USE_CUDA "Whether to build CUDA accelerated FINUFFT library (libcufinufft). This is completely independent of the main FINUFFT library" OFF)
 option(FINUFFT_STATIC_LINKING "If ON builds the static finufft library, if OFF build a shared finufft library." ON)
+option(FINUFFT_POSITION_INDEPENDENT_CODE "Whether to build the finufft library with position independent code (-fPIC). This is only effective when FINUFFT_STATIC_LINKING is ON." ON)
```

Comment:
default OFF.
And edit explanation to say SHARED forces this to be ON.

Reply:
After discussion, default ON.  Power users can experiment with OFF but ON is more likely to work in moer situations.

Reply:
Add to docs/devnotes.rst that devs may want switch off for performance.